ui/src/ensembler/list/ListEnsemblersTable.js

```diff
@@ -115,11 +115,11 @@ export const ListEnsemblersTable = ({
         header: true,
         fullWidth: false,
       },
-      align: "left",
+      align: "right",
       width: "20%",
       render: (id, ensembler) => (
-        <EuiFlexGroup component="div" direction="column" gutterSize="xs" alignItems="flexEnd" >
-          <EuiFlexItem component="div" style={{alignItems: 'flex-start'}}>
+        <EuiFlexGroup component="div" direction="column" gutterSize="s" alignItems="flexEnd" >
+          <EuiFlexItem component="div" style={{alignItems: 'flex-end', gap:'8px'}}>
```

Comment:
Hahah if you're still around @zenovore, here's a quick question: 
- why is `gutterSize` changed from `xs` to `s` (the `ListEnsemblingJobsTable` doesn't feature this change 😅)? 
- why is the `gap` style introduced here? 😅

Feel free to ignore this message if you are busy; just wondering if you happened to remember why those changes were made ahahha

Reply:
> why is gutterSize changed from xs to s (the ListEnsemblingJobsTable doesn't feature this change 😅)?

it was to increase the button size, yes i'm sorry, the `ListEnsemblingJobsTable` should have this change also 🙏
> why is the gap style introduced here? 😅

the gap is giving more space between button, since we change the alignment

Reply:
Ahh got it! xD I see the differences now after finally testing the changes! :D Thanks a lot for updating the PR and for responding so quickly! 🙏🙏